docs/Style-Guide.md

```diff
@@ -68,6 +69,33 @@ When developing any new feature, make sure to add or update the respective tests
 
 When writing E2E tests, follow [Cypress Best Practices](https://docs.cypress.io/guides/references/best-practices.html).
 
+
+## Feedback Messages
+
+When the user performs an action (ex: submiting a form), it's expected to show them some type of feedback, when succeedded or when something went wrong. Here's a guide of the current type of messages we show to them:
+
+### Informative / Success messages
+
+> Your changes were saved!  
+
+### Error messages
+
+#### While filling a form...
```

Comment:
I don't find these sections to be very helpful personally. I think it would be more useful to newcomers to simply show them the API you've designed, with a few examples from the actual code, in two sections (one for scoped banner usage, and one for global banner usage).